store/daystore/day_store.go

```diff
@@ -79,7 +79,8 @@ func NewDayStore(epsilon uint64, resolution time.Duration) *DayStore {
 	// Calculate how many resolutions correspond to an hour
 	hourNS := time.Hour.Nanoseconds()
 	resNS := resolution.Nanoseconds()
-	intervals := uint8(hourNS / resNS)
+	intervals := uint16(hourNS / resNS)
```

Comment:
I opted to use the next largest possible type than uint8. I am not very familiar with Go's struct padding so I am not sure if that will result in any space optimization .

If you believe that uint32 is more appropriate, I'd gladly change it